### PR TITLE
MGMT-15753 Download installation logs button is disabled when destroying cluster

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx [hunk 1]
```diff
@@ -45,11 +45,25 @@ export function ClusterDestroy(props: { isLoading: boolean; cluster: Cluster })
       isLoading={props.isLoading}
       loadingTitle={loadingTitle}
       loadingMessage={
-        <Trans
-          i18nKey={`${props.cluster?.status}.inprogress.message`}
-          values={{ clusterName: props.cluster?.displayName }}
-          components={{ bold: <strong /> }}
-        />
+        props.cluster?.status === ClusterStatus.destroying && isHybrid ? (
+          // t('destroying.ai.inprogress.message')
```
Comment (mareklibra) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx:49]: leftovers

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx [hunk 2]
```diff
@@ -45,11 +45,25 @@ export function ClusterDestroy(props: { isLoading: boolean; cluster: Cluster })
       isLoading={props.isLoading}
       loadingTitle={loadingTitle}
       loadingMessage={
-        <Trans
-          i18nKey={`${props.cluster?.status}.inprogress.message`}
-          values={{ clusterName: props.cluster?.displayName }}
-          components={{ bold: <strong /> }}
-        />
+        props.cluster?.status === ClusterStatus.destroying && isHybrid ? (
```
Comment (mareklibra) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx:48]: nit: The only thing changing here is the `i18nKey`. What about calculating it a few lines above before the `return` statement?

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx (hunk 1 above)
Comment (rawagner) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx:49]: this is not a leftover. It is for the i18n parser to pick up the translation key below as it is constructed programatically

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx (hunk 2 above)
Comment (rawagner) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx:48]: will do!

Comment (rawagner) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterDestroy.tsx]: fixed

**Review by mareklibra — APPROVED** [review]

**Review by ammont82 — APPROVED** [review]

**Pull request merged**